openvpn/parameters/defaults.yaml

```diff
@@ -13,11 +14,12 @@ openvpn:
   group: nobody
   # None, will default to 'user'
   log_user: ~
-  multi_services: false
+  multi_services: {{ salt['grains.has_value']('systemd') }}
```

Comment:
So I can see this Jinja is being used to reconcile this line in the existing `map.jinja`:

https://github.com/saltstack-formulas/openvpn-formula/blob/2e54ec4acdc57e88ef6c28c6d7bbf6e813c53e2f/openvpn/map.jinja#L6

However, one of the advantages of this new `map.jinja` solution is that we can remove so much of the Jinja that is being used in our YAML files across the org, which is actually a syntax error (from a `yamllint` perspective) and a code smell (in that all Jinja-based processing really should be in one of the actual Jinja or state files).

Based on my general experience with the `map.jinja` files that are used across the org, most conversion problems could be avoided if our new `map.jinja` allowed any number of "miscellaneous" Jinja statements to be run in two places: `before` and `after` the map merging process takes place.  So this formula has an example of pre-processing (`before`).  Many formulas (including the `template-formula`) have examples of post-processing (`after`).  Ultimately, since this new `map.jinja` implementation is virtually identical across formulas, it will be really easy to manage from the `ssf-formula`, allowing this extra Jinja to be available anywhere we need it to be.

Do you think this would be possible?  If it can be done, then we wouldn't need to disable `yamllint` for the `defaults.yaml` file in the first place!

Reply:
`map.jinja` only load `.yaml` files since we explicitly use [`load_yaml`](https://github.com/saltstack-formulas/openvpn-formula/pull/134/files#diff-952fc3e054b577be92939c365822ecc581fec4eaa57b736c4a23d574dd53684fR223).

Maybe we could modify [the constraint](https://github.com/saltstack-formulas/openvpn-formula/pull/134/files#diff-952fc3e054b577be92939c365822ecc581fec4eaa57b736c4a23d574dd53684fR206-R216) to load whatever file extension but process the loaded file as `YAML`?

Reply:
Now, each `.yaml` file load is followed by the load of a `.yaml.jinja` and a final `post-map.jinja` permit to post-process the `mapdata` variable.